### PR TITLE
Listen to saptune update broadcast messages in frontend

### PR DESCRIPTION
**Pull request by arbulu89**
# Description

Listen to `saptune_status_updated` broadcast messages and update redux store.

Depends on: https://github.com/trento-project/web/pull/1821

## How was this tested?

UT

### CODE REVIEW

**Review by CDimonaco — APPROVED** [review]
Just a minor comment, LGTM

File: assets/js/components/HostDetails/HostDetails.jsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ function HostDetails({
     package_version: saptuneVersion,
     configured_version: saptuneConfiguredVersion,
     tuning_state: saptuneTuning,
-  } = saptuneStatus;
+  } = saptuneStatus || {};
```
[the remark this answers is not in the document]
Comment (arbulu89) [reply assets/js/components/HostDetails/HostDetails.jsx:61]: Unfortunately it is need, as the `status` might come as null.

[the remark this answers is not in the document]
Comment (EMaksy) [reply assets/js/components/HostDetails/HostDetails.jsx:61]: Passing it from `HostDetailPage` as prop `host.saptune_status || {}` would spair us to have  two times the defaul value in `HostDetails` 
we could also declare `const saptuneStatus = host.saptune_status || {};`  and pass it from `HostDetailPage`

Or what do you think ? :D

**Review by EMaksy — APPROVED** [review]
Hey nice job, I just left a small comment but it's fully up to you :D
Feel free to merge, LGTM!

File: assets/js/components/HostDetails/HostDetails.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (arbulu89) [reply assets/js/components/HostDetails/HostDetails.jsx:61]: alright, i will do the "or" operation in the `Page` view

**Pull request merged**